docs/db-functions.md

```diff
@@ -229,6 +229,19 @@ syntax similar to the `select` family of functions, but returns a Boolean value:
 (db/exists? User :first-name "Spam") ; -> false
 ```
 
+### select-reducible
+
+Using `select` will realize the full set of results in memory. For smaller sets of results this is fine but queries that
+could potentially return many rows, this could cause memory issues. `select-reducible` is an similar function to `select`,
+but will return a reducible sequence instead of a vector. Using this, it's possible to consume the query results as they
+are streamed from the database. Using this, you can avoid fully realizing the set of results in memory.
+
+```
+(transduce (filter complex-filter-logic-fn) serialize-to-http-response output-stream (db/select-reducible User :active true))
+```
```

Comment:
Can we 

1) Add `clj` after the triple-backtick to get syntax highlighting for the example
2) break the example up into multiple lines so we can see it without needing vertical scrollbars

Reply:
since everything else looks good I'll just go ahead and fix this